content/applications/hr/payroll/payslips.rst

```diff
@@ -28,159 +30,235 @@ from this dashboard.
+  Hours`, and :guilabel:`Amount`) are automatically filled in, based on what was entered for the
+  :guilabel:`Period`, :guilabel:`Contract`, and :guilabel:`Structure` fields of the payslip form.
+- :guilabel:`Other Inputs`: additional inputs affecting the payslip can be entered in this section,
+  such as deductions, reimbursements, and expenses.
+
+  Click :guilabel:`Add a line` to create an entry in the :guilabel:`Other Inputs` section.
+
+  Using the drop-down menu in the :guilabel:`Type` column, select a :guilabel:`Type` for the input.
+  Next, enter a :guilabel:`Description`, if desired. Lastly, enter the amount in the
+  :guilabel:`Count` field.
+
+.. image:: payslips/worked-days-tab.png
+   :align: center
+   :alt: The fields filled out in the worked days and inputs tab.
+
+Salary computation tab
+**********************
+
+- :guilabel:`Salary Computation`: the :guilabel:`Salary Computation` tab is automatically filled in
+  after the :guilabel:`Compute Sheet` button is clicked. Doing so displays the entry's wages,
   deductions, taxes, etc. for the entry.
-- :guilabel:`Batch Name`: Located in the :guilabel:`Accounting Information` tab, select the payslip
-  batch this new payslip should be added to from the drop-down menu.
-- :guilabel:`Date Account`: Located in the :guilabel:`Accounting Information` tab, enter the date on
-  which the payslip should be posted, by clicking on the drop-down menu, and navigating to the
-  correct month and year using the :guilabel:`< > (less-than/greater-than)` icons in the calendar
-  pop-up window. Then, click on the desired date.
-- :guilabel:`Salary Journal`: This field, located in the :guilabel:`Accounting Information` tab,
-  represents the journal that the payslip will be logged in, and is automatically selected when the
-  :guilabel:`Contract` and :guilabel:`Structure` are entered in the payslip form.
-- :guilabel:`Accounting Entry`: This field, located in the :guilabel:`Accounting Information` tab,
-  is automatically filled in once the payslip is confirmed.
-
-Save and process new payslip
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When all the necessary information on the payslip is entered, click :guilabel:`Save` to save the
-data, or click :guilabel:`Discard` to delete the entry.
+- :guilabel:`Has Negative Net To Report`: click the checkbox if the employee has a negative net
+  amount for this payslip. This **only** appears if the employee's payslip has a negative balance.
 
-.. note::
-   Saving the entry is not required in order to compute the sheet, although it is considered best
-   practice to do so. The :guilabel:`Compute Sheet` button can be clicked without first saving the
-   payslip. Doing so will save the entry *and* compute the sheet.
+.. image:: payslips/salary-comp-tab.png
+   :align: center
+   :alt: The fields filled out in the salary computation tab.
+
+Other info tab
+**************
+
+- :guilabel:`Payslip Name`: in the :guilabel:`Other Info` tab, type in a name for the payslip in
```

Comment:
```suggestion
- :guilabel:`Payslip Name`: type in a name for the payslip in
```
since it's already in the Other Info tab section, I don't believe the second mention of its tab whereabouts is necessary